golem-cli/src/main.rs

```diff
@@ -187,6 +189,32 @@ async fn async_main(cmd: GolemCommand) -> Result<(), Box<dyn std::error::Error>>
         templates: &template_srv,
     };
 
+    let health_check_client_for_template = HealthCheckClientLive {
+        client: golem_client::api::HealthCheckClientLive {
+            context: template_context.clone(),
+        },
+    };
+
+    let health_check_client_for_worker = HealthCheckClientLive {
+        client: golem_client::api::HealthCheckClientLive {
+            context: worker_context.clone(),
+        },
+    };
+
+    let update_srv = VersionHandlerLive {
+        template_client: health_check_client_for_template,
+        worker_client: health_check_client_for_worker,
+    };
+
+    let yellow = "\x1b[33m";
+    let reset_color = "\x1b[0m";
+
+    let version_check = update_srv.check().await;
+
+    if let Err(err) = version_check {
+        println!("{}{}{}", yellow, err.0, reset_color)
```

Comment:
```suggestion
        eprintln!("{}{}{}", yellow, err.0, reset_color)
```

should use stderr, because stdout can be used programmatically

Reply:
done, thanks!